cmd/ipfs/daemon.go

```diff
@@ -278,6 +279,12 @@ func serveHTTPApi(req cmds.Request) (error, <-chan error) {
 			},
 		},
 	})
+
+	metricsOption := func(n *core.IpfsNode, mux *http.ServeMux) (*http.ServeMux, error) {
+		mux.Handle("/metrics", prom.Handler())
+		return mux, nil
+	}
```

Comment:
wonder if we should be putting all these routes (eventlog, etc) under something other than root. maybe

```
/debug/logs
/debug/metrics
```
or 
```
/sys/logs
/sys/metrics
```
or something